portfolio/src/main/webapp/index.html

```diff
@@ -7,55 +7,41 @@
     <script src="script.js"></script>
   </head>
   <body onload="getMessages()">
-    <div id ="webHeader">
-    <h1 >Jordan's Portfolio</h1>
-    </div>
-
-    <div id = "logo">
-        <a href="https://www.youtube.com/watch?v=ICEqywt012k">
-        <img src="img/jordan.jpeg">
-        </a>
-    </div>
-
     <div id="content">
-            <h1>Welcome to my corner of the internet !</h1>
-        <div class="header">
-            <h2> "A Little Introduction"</h2>
+     <div class="content-container">
+       <div class="content-left"></div>
+        <div class="content-center">
```

Comment:
It seems like these two should be the same indentation level?

Reply:
Ack